dev/PowerNotifications/PowerNotifications.h

```diff
@@ -649,14 +604,16 @@ namespace winrt::Microsoft::ProjectReunion::factory_implementation
         }
 
     };
-
 };
 
 namespace winrt::Microsoft::ProjectReunion::implementation
 {
+    // EffectivePowerMode variables
+    static ULONG g_powerModeVersion = NULL;
```

Comment:
What locking mode is required/desired here? Should both of these be `std::atomic<T>` so they can be safely updated/read? Or should there be a `std::mutex` that can be locked while mutating/reading them?

Reply:
I don't think any locking is desired; but have still changed it to atomic. There is an unlikely scenario that two threads will check for the version, which will result in checking the version again instead of using this cached value. Is there any other scenario that I might have missed?